Lacework/tests/client/test_auth.py

```diff
@@ -0,0 +1,103 @@
+import os
+from datetime import datetime, timedelta
+
+import pytest
+import requests_mock
+
+from lacework_module.client.auth import LaceworkAuthentication
+
+
+def test_get_credentials():
+    lacework_url="api"
+    access_key="foo"
+    secret_key="bar"
+    auth = LaceworkAuthentication(lacework_url, access_key, secret_key)
+
+    with requests_mock.Mocker() as mock:
+        mock.register_uri(
+            "POST",
+            url=f"https://{lacework_url}.lacework.net/api/v2/access/tokens",
+            headers={
+                    "X-LW-UAKS": secret_key,
+                    "Content-Type": "application/json"
+                },
+            json={
+                    "token": "foo-token",
+                    "expiresAt": str(datetime.utcnow() + timedelta(seconds=3600))
```

Comment:
This date should be ISO8601 formatted:
```suggestion
                    "expiresAt": (datetime.utcnow() + timedelta(seconds=3600)).isoformat()
```